test/token/ERC20/MintableToken.behaviour.js

```diff
@@ -0,0 +1,148 @@
+import assertRevert from '../../helpers/assertRevert';
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+export default function ([owner, anotherAccount, minter]) {
+  describe('as a basic mintable token', function () {
```

Comment:
I'm going to merge these changes, but in the future, there's a `context` helper from Mocha for this feature; it's the same thing as describe, but helps the reader separate a "context" from the description of what the thing should do